Index: src/main/java/fcul/viegas/topologies/machinelearning/method/Topologies_EVALUATION_DISTRIBUTED_HYBRID_CASCADE_WithConformal.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/main/java/fcul/viegas/topologies/machinelearning/method/Topologies_EVALUATION_DISTRIBUTED_HYBRID_CASCADE_WithConformal.java	(revision 01ddb6711e9aac8aed4a463fef1b065d18e9df84)
+++ src/main/java/fcul/viegas/topologies/machinelearning/method/Topologies_EVALUATION_DISTRIBUTED_HYBRID_CASCADE_WithConformal.java	(date 1517226302000)
@@ -238,8 +238,8 @@
         oos.close();
 
         ArrayList<String[]> testFiles = new ArrayList<>();
-        for (int i = 0; i < testFilesVIEGAS.size(); i++) {
-        //for (int i = 0; i < 1000; i++) {
+        //for (int i = 0; i < testFilesVIEGAS.size(); i++) {
+        for (int i = 0; i < 600; i++) {
 
             String[] array = new String[4];
             array[0] = testFilesVIEGAS.get(i);
Index: src/main/java/fcul/viegas/topologies/machinelearning/MachineLearningModelBuilders.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/main/java/fcul/viegas/topologies/machinelearning/MachineLearningModelBuilders.java	(revision 01ddb6711e9aac8aed4a463fef1b065d18e9df84)
+++ src/main/java/fcul/viegas/topologies/machinelearning/MachineLearningModelBuilders.java	(date 1517226371000)
@@ -520,8 +520,13 @@
     //funcionando teoricamente...
     public moa.classifiers.AbstractClassifier trainClassifierOzaBaggingMOA(Instances train) throws Exception {
         OzaBag classifier = new OzaBag();
+
         classifier.ensembleSizeOption = new IntOption("ensembleSize", 's',
                 "The number of models in the bag.", 20, 1, Integer.MAX_VALUE);
+        classifier.baseLearnerOption = new ClassOption("baseLearner", 'l',
+                "Classifier to train.", moa.classifiers.trees.HoeffdingTree.class,
+                "moa.classifiers.trees.HoeffdingTree -g 201");
+
 
         WekaToSamoaInstanceConverter converter = new WekaToSamoaInstanceConverter();
         com.yahoo.labs.samoa.instances.Instances moaTrain = converter.samoaInstances(train);
